src/i18n/en-US.properties

```diff
@@ -54,6 +54,12 @@ error_file_type_unsupported=We're sorry, this file format is not supported.
 error_iwork=We're sorry, iWork files are not currently supported.
 # Preview error message shown when document loading fails (most likely due to password or watermark)
 error_document=We're sorry, the preview didn't load. This document may be protected.
+# Preview error message shown when the document is password protected
+error_password_protected=We're sorry the preview didn't load. This document is protected.
+# Preview error message shown when conversion was unable to process the file at the given time.
+error_try_again_later=We're sorry the preview didn't load. Please try again later.
+# Preview error message shown when conversion failed due to file contents
+error_unsupported_format=We're sorry the preivew didn't load. This file may be corrupted.
```

Comment:
Is this supposed to be `error_unsupported_format`? Or should it be changed to `error_corrupted_file`?

Reply:
good point. Harris is going to look into all of the cases that cause this error to fire.

Reply:
This applies to a variety of situations including both unsupported files and corrupted ones. Technically a corrupted file is unsupported so that's slightly more of an umbrella term?

Reply:
Also, spelling: 'preview' not 'preivew'